test/e2e/README.md

```diff
@@ -32,7 +32,10 @@ The first step to running the e2e tests is setting up the required environment v
 | `VSPHERE_NETWORK`             | The unique name or inventory path of the network to which VMs will be connected                       | `my-network` or `/my-datacenter/network/my-network`                              |
 | `VSPHERE_HAPROXY_TEMPLATE`    | The unique name or inventory path of the template from which the HAProxy load balancer VMs are cloned | `my-haproxy-template` or `/my-datacenter/vm/my-haproxy-template`                 |
 | `VSPHERE_SSH_PRIVATE_KEY`     | The file path of the private key used to ssh into the CAPV VMs                                        | `/home/foo/bar-ssh.key`                                                          |
-| `VSPHERE_SSH_AUTHORIZED_KEY`  | The public key that is added to the CAPV VMs                                                          | `ss-rsa ABCDEF...XYZ=`                                                          |
+| `VSPHERE_SSH_AUTHORIZED_KEY`  | The public key that is added to the CAPV VMs                                                          | `ssh-rsa ABCDEF...XYZ=`                                                          |
```

Comment:
Could you also remove the `VSPHERE_HAPROXY_TEMPLATE` on line 33, that is something that is not supported anymore.

Reply:
Done.